src/main/java/io/cdap/plugin/spark/HierarchyToRelationalConfig.java

```diff
@@ -134,12 +134,12 @@ public void validate(FailureCollector collector) {
     }
     if (!Strings.isNullOrEmpty(PARENT_CHILD_MAPPING_FIELD)) {
       Map<String, String> parentChildMapping = getParentChildMapping();
-      if (parentChildMapping.containsKey(parentField)) {
+      if (parentChildMapping.containsKey(parentField) || parentChildMapping.containsValue(parentField)) {
         collector.addFailure("Parent key field found mapping.",
```

Comment:
This is for indicating that one of the fields in Parent->Child mapping is the Parent Field - which should not be included as it would result in the following issue ex:
Config:
``parentField="ParentId"``
``parentChildMappingField="ParentId=ChildId;"``
resulting sql:
``select ParentId, ChildId, ParentId ...`` <- cannot have same column twice on the sql and also there is already seperate config for ParentId->ChildId mapping.